src/run.js

```diff
@@ -69,19 +70,47 @@ const minimalcss = async options => {
 
     // To build up a map of all downloaded CSS
     page.on('response', response => {
-      const url = response.url
+      const responseUrl = response.url
       const ct = response.headers['content-type'] || ''
       if (!response.ok) {
-        throw new Error(`${response.status} on ${url}`)
+        throw new Error(`${response.status} on ${responseUrl}`)
       }
-      if (ct.indexOf('text/css') > -1 || /\.css$/i.test(url)) {
+      if (ct.indexOf('text/css') > -1 || /\.css$/i.test(responseUrl)) {
         response.text().then(text => {
           const ast = csstree.parse(text, {
-            parseValue: false,
+            parseValue: true,
             parseRulePrelude: false
           })
-          stylesheetAstObjects[url] = csstree.toPlainObject(ast)
-          stylesheetContents[url] = text
+          csstree.walk(ast, (node) => {
+            if (node.type === 'Url') {
+              let value = node.value;
+              let path
+              if (value.type === 'Raw') {
+                path = value.value;
+              } else {
+                path = value.value.substr(1, value.value.length - 2);
+              }
+              const absolute = /^https?:\/\/|^\/\//i
+              const root = /^\//
```

Comment:
These two variables don't need to be variables. They're only used once (each) and they're also a bit "poorly named". Like `absolute` doesn't carry any descriptive value. Unlike "absoluteUrl", "nodeTree" or "tree". Perhaps they should be called "absoluteUrlRegex"?

Can you consider putting the regexes inline?
For example:
```javascript
if (/^https?:\/\/|^\/\//i.test(path)) {
  ...
```